### PR TITLE
554 refactor and fix availability spec

### PR DESCRIPTION
**Pull request by banukutlu**
Issue #554 

## Additional Work

refactoring blackcat_messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +58,15 @@ Follow the instructions for [How To Use Webpacker](https://github.com/psu-librar
...
+You can add an optional non-tracked (by git) yml file at `config/blackcat_messages.yml` to (1) modify the announcement bar (thin bar at top), (2) put the site in "readonly" (no availability data) with a message flashed as error, (3) to add an ad-hoc alert (flash error message) or (4) to put holds in readonly mode by hiding the I Want It button. 
 
 The keys dictate the behavior:
 
 ```yml
 readonly: 
 announcement:
 alert:
+readonly_holds:
 ```
```
Comment (cdmo) [inline README.md:71]: What do you think about add a line about how to populate this field? 

```suggestion

`readonly_holds` expects a truthy value. `true` or any text here would evaluate to true and hide the I Want It button globally. Not being present at all, having a blank value or `false` all evaluate to `false` and leave the I Want It button in place.

```

**Comment by cdmo** [comment]
Add https://github.com/psu-libraries/psulib_blacklight/issues/557 to milestone .5 x

**Comment by banukutlu** [comment]
@cdmo I changed `blackcat_messages` to `blackcat_admin`, what do you think about renaming the rest, like the should we rename the service class as BlackcatAdmin and any other references?

**Comment by cdmo** [comment]
Can you take a look at #558?

**Pull request merged**